### PR TITLE
Update secret location for CCA tests

### PR DESCRIPTION
**Pull request by neha-bhargava**
Fixes 
Update the location of secret

**Changes proposed in this request**

**Testing**

**Performance impact**

**Documentation**
- [ ] All relevant documentation is updated.

### CODE REVIEW

File: tests/Microsoft.Identity.Test.Integration.netcore/Infrastructure/ConfidentialAppSettings.cs [hunk 1]
```diff
@@ -37,9 +37,9 @@ public class ConfidentialAppSettings
     {
         private class PublicCloudConfidentialAppSettings : IConfidentialAppSettings
         {
-            public string ClientId => UseAppIdUri? "https://request.msidlab.com" : "f62c5ae3-bf3a-4af5-afa8-a68b800396e9";
+            public string ClientId => UseAppIdUri? "https://request.msidlab.com" : "88f91eac-c606-4c67-a0e2-a5e8a186854f";
```
Comment (gladjohn) [inline tests/Microsoft.Identity.Test.Integration.netcore/Infrastructure/ConfidentialAppSettings.cs:40]: this AppIdUri `"https://request.msidlab.com"` is for the lab app `f62c5ae3-bf3a-4af5-afa8-a68b800396e9` in MSFT tenant.

Comment (gladjohn) [reply tests/Microsoft.Identity.Test.Integration.netcore/Infrastructure/ConfidentialAppSettings.cs:40]: the lab app's app id uri : `api://88f91eac-c606-4c67-a0e2-a5e8a186854f`

Comment (neha-bhargava) [reply tests/Microsoft.Identity.Test.Integration.netcore/Infrastructure/ConfidentialAppSettings.cs:40]: What is the name of the cert for this app? LabVaultAccessCert? Does this change for all the apps for cca testing or only the ones using secret?

Comment (gladjohn) [reply tests/Microsoft.Identity.Test.Integration.netcore/Infrastructure/ConfidentialAppSettings.cs:40]: I now added LabVaultAccessCert to this app

**Review by bgavrilMS — APPROVED** [review]

File: tests/Microsoft.Identity.Test.Integration.netcore/Infrastructure/ConfidentialAppSettings.cs (hunk 1 above)
Comment (neha-bhargava) [reply tests/Microsoft.Identity.Test.Integration.netcore/Infrastructure/ConfidentialAppSettings.cs]: updated

File: tests/Microsoft.Identity.Test.Integration.netcore/HeadlessTests/PoPTests.NetFwk.cs [hunk 2]
```diff
@@ -58,7 +58,7 @@ public void TestInitialize()
 
             if (_keyVault == null)
             {
-                _keyVault = new KeyVaultSecretsProvider(KeyVaultInstance.MsalTeam);
+                _keyVault = new KeyVaultSecretsProvider(KeyVaultInstance.MSIDLab);
```
Comment (gladjohn) [inline tests/Microsoft.Identity.Test.Integration.netcore/HeadlessTests/PoPTests.NetFwk.cs:61]: There is a lab API query to get all these details, we should at some point shift to using the query `https://msidlab.com/api/config?publicClient=no&signInAudience=azureadmyorg`

Comment (gladjohn) [inline tests/Microsoft.Identity.Test.Integration.netcore/HeadlessTests/PoPTests.NetFwk.cs:61]: there are only two more secret references after this that points to `KeyVaultInstance.MsalTeam`, please file a work item for us to move those secrets to `msidlabs` vault and remove `KeyVaultInstance.MsalTeam` references

**Review by gladjohn — APPROVED** [review]

File: tests/Microsoft.Identity.Test.Integration.netcore/HeadlessTests/PoPTests.NetFwk.cs (hunk 2 above)
Comment (neha-bhargava) [reply tests/Microsoft.Identity.Test.Integration.netcore/HeadlessTests/PoPTests.NetFwk.cs]: No there are 4. 2 for OBO 2 in Selenium tests

**Review by trwalke — APPROVED** [review]

**Comment by bgavrilMS** [comment]
Ready to merge?

**Comment by neha-bhargava** [comment]
> Ready to merge?

There are still a few more tests failing. Looking into those.

**Review by gladjohn — APPROVED** [review]

**Pull request merged**